src/background/main.ts

```diff
@@ -10,14 +11,16 @@ if (import.meta.hot) {
   import('/@vite/client');
 }
 
-const { toggleWebRTC, webRTCStatus } = useWebRtc();
+const { setWebRTC, webRTCStatus } = useWebRtc();
 const { cleanOutdated } = useRecommendations();
 
 // Add listeners on extension actions
 addExtListeners();
 
 // Load webRTC settings from storage
-toggleWebRTC(webRTCStatus.value);
+watch(webRTCStatus, () => {
```

Comment:
This looks a bit odd...
You are getting both the value and the setter from the same composable `useWebRtc`, could this call be done inside that file instead of doing it here? 
Or is there a particular reason to do this in this file?

Reply:
No particular reason, I just need to have it updated.

So I'm considering whether I should abstract this in an init function I call from the background script or if I should simply declare the watch in the composable as is.

```
watch(webRTCStatus, () => {
  setWebRTC(webRTCStatus.value);
});
```

Do you have any recommendation?

Reply:
The only other file interacting with this composable is `WebRTCToggle.vue`, I'm unsure if there's an opportunity for refactoring here as well.